R/mime_part.R

```diff
@@ -176,9 +176,26 @@ mime_part.character <- function(x, name, ...) {
   if (length(x) == 1 && file.exists(x)) {
     .file_attachment(x, name, ...)
   } else {
-     .mime_part(headers=list(
-                 "Content-Type"="text/plain",
-                 "Content-Disposition"="inline"),
-               text=paste(x, collapse="\r\n"))
+
+    # Default content_type (backward compatibillity)
+    content_type <- "text/plain"
+
+    # If Encoding is set then use this
+    # R will always use ASCII over UTF-8 if possible
+    # ASCII is reported as unknown
+    encoding <- setdiff(Encoding(x), "unknown")
+
+    if (length(encoding == 1))
+      content_type <- paste0("text/plain; charset=", encoding, "; format=flowed")
```

Comment:
Should `format=flowed` really be the default? I think it'd be better to add an additional argument to set the format. Maybe we should just call `mime_part.character` and fix up the headers of the result so that it is consistent?

Reply:
A flowed argument was added to `mime_part.character()`. This defaults to `FALSE`.